packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -25,9 +25,9 @@ Requires: foreman >= %{foreman_min_version}
 BuildRequires: foreman-assets >= %{foreman_min_version}
 BuildRequires: foreman-plugin >= %{foreman_min_version}
 Requires: ruby >= 2.5
-Requires: ruby < 3
+Requires: ruby < 3.1
 BuildRequires: ruby >= 2.5
-BuildRequires: ruby < 3
+BuildRequires: ruby < 3.1
```

Comment:
I suspect we'll raise this to >= 2.7 and < 4.

Reply:
Probably. My goal was "update it to something so it builds, and then when the actual Ruby3 PR is in, we regenerate the deps anyway"